BingeIT/src/App.js

```diff
@@ -6,6 +6,11 @@ import { Header } from "./components/Header/Header";
 import { Sidebar } from "./components/Sidebar/Sidebar";
 import { Footer } from "./components/Footer/Footer";
 import { VideoPage } from "./pages/VideoPage/VideoPage";
+import { LoginPage } from "./pages/AuthPage/Loginpage";
+import { Profile } from "./pages/AuthPage/Profile";
+import { PrivateRoute } from "./routes/PrivateRoute";
+import { Playlists } from "./pages/PlaylistsPage/PlaylistsPage";
+import PlaylistPage from "./pages/PlaylistPage/PlaylistPage";
```

Comment:
You can create index file and export there all file so you have to only here import using one line

Reply:
okay